.vscode/settings.json

```diff
@@ -0,0 +1,38 @@
+{
+  "python.linting.enabled": true,
+  "python.linting.pylintEnabled": true,
+  "python.linting.flake8Enabled": false,
+  "python.linting.pycodestyleEnabled": false,
+  "python.formatting.provider": "yapf",
+  "python.formatting.yapfPath": "yapf",
+  "python.formatting.yapfArgs": ["--style", "${workspaceRoot}/setup.cfg"],
+  "cSpell.words": [
+    "AGRC",
+    "agol",
+    "arcgis",
+    "dotenv",
+    "getenv",
+    "instafail",
+    "isort",
+    "mssql",
+    "neighbour",
+    "pgsql",
+    "psycopg",
+    "pygsheets",
+    "pylint",
+    "pyodbc",
+    "pytest",
+    "rindex",
+    "sgid",
+    "yapf"
+  ],
+  "editor.formatOnSave": true,
+  "editor.rulers": [120],
+  "coverage-gutters.showGutterCoverage": false,
+  "coverage-gutters.showLineCoverage": true,
+  "coverage-gutters.showRulerCoverage": false,
+  "coverage-gutters.highlightdark": "rgb(61, 153, 112, .05)",
+  "coverage-gutters.noHighlightDark": "rgb(255, 65, 54, .05)",
+  "coverage-gutters.partialHighlightDark": "rgb(255, 133, 27, .05)",
+  "python.pythonPath": "/Users/sgourley/dev/porter/.env/bin/python3"
```

Comment:
I think that VSCode supports removing this project setting now. It's stored somewhere else.

Reply:
https://github.com/microsoft/vscode-python/wiki/AB-Experiments